lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1922,6 +1922,24 @@ Value MemOp::getPortNamed(StringAttr name) {
   return Value();
 }
 
+static Value trackValueThroughAnoymousWires(Value value) {
+  auto wire = value.getDefiningOp<WireOp>();
```

Comment:
Nodes and wires are basically interchangable, we should track through them too I think

Reply:
Node is also tracked via `getModuleScopeDriver`.